internal/infra/updater_test.go

```diff
@@ -0,0 +1,50 @@
+package infra
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func Test_mountOptions(t *testing.T) {
+	wd, _ := os.Getwd()
+	tests := []struct {
+		input            string
+		expectedLocal    string
+		expectedRemote   string
+		expectedReadOnly bool
+		expectedErr      error
+	}{
+		{
+			input:       "",
+			expectedErr: ErrInvalidVolume,
+		}, {
+			input:          "local:remote",
+			expectedLocal:  filepath.Join(wd, "local"),
+			expectedRemote: "remote",
+		}, {
+			input:            "local:remote:ro",
+			expectedLocal:    filepath.Join(wd, "local"),
+			expectedRemote:   "remote",
+			expectedReadOnly: true,
+		}, {
+			input:            ".:remote:ro",
+			expectedLocal:    wd,
+			expectedRemote:   "remote",
+			expectedReadOnly: true,
+		}, {
+			input:       "local:remote:ro:hi",
+			expectedErr: ErrInvalidVolume,
+		}, {
+			input:       "local:remote:wo",
+			expectedErr: ErrInvalidVolume,
+		},
+	}
+
+	for _, test := range tests {
+		local, remote, readOnly, err := mountOptions(test.input)
+		if local != test.expectedLocal || remote != test.expectedRemote || readOnly != test.expectedReadOnly || err != test.expectedErr {
+			t.Errorf("For input '%v' got '%v' '%v' '%v' '%v'", test.input, local, remote, readOnly, err)
```

Comment:
I guess I'm late to the party, but do we not use `testify` in the `cli`?

It's so convenient so many places, and such a popular lib in the ecosystem that I've got no problems with adding it as a test-only dep.

Reply:
A few reasons I'm avoiding testify for now:

- Trying to adopt the behaviors and patterns of the language I'm working in, the Go developers recommend avoiding dependencies.
- testify's strength seems to be in presenting nice diffs, I haven't felt that need here yet
- I've seen references to "don't use this or that feature" in testify and I don't want to have to learn what that means yet, if statements work pretty well
- Copilot generates these checks perfectly so it's not a big burden to type out anymore! 🙌 
- It's easier to leave it out now than to remove it later